woocommerce/wc-admin
# Segmented Selection: re-apply borders for all browsers

Borders were moved to `ie.scss` in https://github.com/woocommerce/wc-admin/pull/1199/files. This PR moves the CSS rules back to the main stylesheets as those rules apply to all browsers.

### Before

<img width="345" alt="screen shot 2019-02-14 at 12 39 55 pm" src="https://user-images.githubusercontent.com/1922453/52752004-41c10400-3056-11e9-9a19-0fb0b4270f86.png">

### After

<img width="337" alt="screen shot 2019-02-14 at 12 40 50 pm" src="https://user-images.githubusercontent.com/1922453/52752013-4685b800-3056-11e9-9d78-34fe89be58db.png">

### Detailed test instructions:

1. Go to any date related report, such as Products.
2. Open the Datepicker and see borders on preset options, eg "Today", "Yesterday".
